DevelopGitHub/GitHubExtension/GitHubExtension.Infrastructure.Constants/RouteConstants.cs

```diff
@@ -9,6 +9,8 @@ public static class RouteConstants
         public const string ApiUser = "api/user";
 
         // static segments
+        public const string Home = "http://localhost:50859/";
```

Comment:
No, it's not secret port number. This port we use for reload page after authentication. And we can't choose this.